platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/address/AddressBookUtils.java

```diff
@@ -356,4 +389,60 @@ public static String formatConsensusEventStreamName(
             return String.valueOf(selfId);
         }
     }
+
+    /**
+     * Extracts the bytes of the given signing certificate. If the certificate is null, an empty byte
+     * array is returned.
+     *
+     * @param signingCert the signing certificate
+     * @return the bytes of the signing certificate wrapped in a {@link Bytes} object
+     */
+    public static Bytes extractSigCertBytes(@Nullable final X509Certificate signingCert) {
+        Bytes signingCertificateBytes;
+        try {
+            signingCertificateBytes = signingCert == null ? Bytes.EMPTY : Bytes.wrap(signingCert.getEncoded());
+        } catch (final CertificateEncodingException e) {
+            signingCertificateBytes = Bytes.EMPTY;
+        }
+
+        return signingCertificateBytes;
+    }
+
+    /**
+     * Maps a {@link Network} object to its equivalent {@link Roster}.
+     *
+     * @param network the network to represent as a roster
+     * @return the converted roster
+     */
+    public static Roster fromNetwork(@NonNull final Network network) {
```

Comment:
nitpick: I would've looked first for such method (and the other two below) in `RosterUtils` instead of here, we can perhaps move their definition there?

Reply:
Good idea, done. Thanks!